doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -396,9 +384,9 @@ Field        | Type       | Meaning
 `worker_id`  | [`bigint`] | The ID of the worker thread hosting the operator.
 `elapsed_ns` | [`bigint`] | The total elapsed time spent in the operator in nanoseconds.
 
-### `mz_raw_compute_operator_durations`
+### `mz_compute_operator_durations_histogram`
 
-The `mz_raw_compute_operator_durations` source stores a histogram describing the
+The `mz_compute_operator_durations_histogram` view describes a histogram describing the
```

Comment:
"describes a histogram describing ..." is a bit awkward. In other places we say "describes a histogram of..."